nsv13/code/modules/overmap/physics.dm

```diff
@@ -32,25 +39,52 @@
 	for(var/turf/T in locs)
 		T.SpinAnimation()
 
+/obj/effect/overmap_hitbox_marker
+	name = "Hitbox display"
+	icon = 'nsv13/icons/overmap/default.dmi'
+	icon_state = "hitbox_marker"
+
+/obj/effect/overmap_hitbox_marker/Initialize(mapload, pixel_x, pixel_y, pixel_z, pixel_w)
+	. = ..()
+	src.pixel_x = pixel_x
+	src.pixel_y = pixel_y
+	src.pixel_z = pixel_z
+	src.pixel_w = pixel_w
+
+//Method to show the hitbox of your current ship to see if youve set it up correctly
+/obj/structure/overmap/proc/display_hitbox()
+	if(!collision_positions.len)
+		return
+
+	for(var/datum/vector2d/point in collision_positions)
+		var/obj/effect/overmap_hitbox_marker/H = new(src, point.x, point.y, abs(pixel_z), abs(pixel_w))
+		vis_contents += H
+
 /obj/structure/overmap/Initialize()
 	. = ..()
 	var/icon/I = icon(icon,icon_state,SOUTH) //SOUTH because all overmaps only ever face right, no other dirs.
 	pixel_collision_size_x = I.Width()
 	pixel_collision_size_y = I.Height()
-//	bound_width = pixel_collision_size_x
-//	bound_height = pixel_collision_size_y
+	offset = new /datum/vector2d()
+	last_offset = new /datum/vector2d()
+	position = new /datum/vector2d(x*32,y*32)
+	velocity = new /datum/vector2d(0, 0)
+	overlap = new /datum/vector2d(0, 0)
+	if(collision_positions.len)
+		collider2d = new /datum/shape(position, collision_positions, angle) // -TORADIANS(src.angle-90)
+	else
+		message_admins("[src] does not have collision points set! It will float through everything.")
 
 /obj/structure/overmap/proc/can_move()
 	return TRUE //Placeholder for everything but fighters. We can later extend this if / when we want to code in ship engines.
 
 /obj/structure/overmap/process(time)
-	time /= 10 // fuck off with your deciseconds
+	time /= 1 SECONDS
```

Comment:
What the HECK IS THIS DOING HERE!


Okay after this PR YOU MUST REWRITE ALL OF THIS TO PROPERLY USE A SUBSYSTEM.

Reply:
...?
This process() is evidence that it already does use a subsystem, overmap processing.